src/tasks/import-data/import-data-v2/index.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as tl from 'azure-pipelines-task-lib/task';
+import { dataImport } from "@microsoft/powerplatform-cli-wrapper/dist/actions";
+import { BuildToolsRunnerParams } from "../../../host/BuildToolsRunnerParams";
+import { getCredentials } from "../../../params/auth/getCredentials";
+import { isRunningOnAgent } from '../../../params/auth/isRunningOnAgent';
+import * as taskDefinitionData from "./task.json";
+import { TaskParser } from "../../../parser/TaskParser";
+import { AzurePipelineTaskDefiniton } from 'src/parser/AzurePipelineDefinitions';
+import { BuildToolsHost } from "../../../host/BuildToolsHost";
+
+(async () => {
+  if (isRunningOnAgent()) {
+      await main();
+  }
+})().catch(error => {
+  tl.setResult(tl.TaskResult.Failed, error);
+});
+
+
+export async function main(): Promise<void> {
+  const taskParser = new TaskParser();
+  const parameterMap = taskParser.getHostParameterEntries((taskDefinitionData as unknown) as AzurePipelineTaskDefiniton);
+
+  await dataImport({
+    credentials: getCredentials(),
+    dataDirectory: parameterMap['DataDirectory'],
+    verbose: parameterMap['Verbose'],
```

Comment:
for now, pls remove the verbose param from task.json and the call to the cli-wrapper action.
Dumping the cmt log into stdout is a nightmare for users to scroll through. Better: in a next PR, re-introduce the verbose task input property, but then add a similar logic as we do for e.g. deploy-package: the task manages the location and name of the verbose output, and at the end uploads the verbose log file into the AzDO artifact store (or adds as sep. file in the case of rel pipelines). This will also need a change in the pac CLI